vagrant/provisioning/roles/arkcase-cognito/tasks/main.yml

```diff
@@ -21,4 +21,12 @@
   template:
     src: "{{ role_path }}/../arkcase-app/templates/arkcase-portal-cognito-server.yaml"
     dest: "{{ root_folder }}/app/arkcase-cognito/application.yml"
-    backup: yes
\ No newline at end of file
+    backup: yes
+
+- name: setup configuration file
```

Comment:
Is this setup also done in the task above this one, which calls the ../arkcase-app/templates/arkcase-portal-cognito-server.yaml template?

Reply:
No, the database settings are not done in that file. I wonder if it wouldn't be ideal to merge both files and keep them in the configuration folder.

Reply:
Merging the files would be great if you have time to do it.  Good catch.